controllers/util/storageconsumer.go

```diff
@@ -13,7 +13,9 @@ import (
 
 const (
 	// Reserved RadosNamespaceName for internal use and their representation at different layes
-	ImplicitRbdRadosNamespaceName = "<implicit>"
+	ImplicitRbdRadosNamespaceName                  = "<implicit>"
+	StorageConsumer418ProcessedAnnotationKey       = "ocs.openshift.io/storageconsumer-4-18-processed"
```

Comment:
The annotation is a way for us to declare the we allow 4.19 logic on this consumer, the name should reflect that
```suggestion
	 Unblock419ReconcileAnnotationKey  = "ocs.openshift.io/unblock-4_19-reconcile
```